### PR TITLE
docs: fix links

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

update links for LTS

<details>
  <summary><b>Before submitting</b></summary>

- Was this **discussed/agreed** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- Did you make sure to **update the documentation** with your changes? (if necessary)
- Did you write any **new necessary tests**? (not for typos and docs)
- [x] Did you verify new and **existing tests pass** locally with your changes?
- Did you list all the **breaking changes** introduced by this pull request?
- Did you **update the CHANGELOG**? (not for typos, docs, test updates, or minor internal changes/refactors)

</details>

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/Lightning-AI/lightning/wiki/Review-guidelines). In short, see the following bullet-list:

<details>
  <summary>Reviewer checklist</summary>

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

</details>

cc @borda @carmocca @justusschock @awaelchli

### CODE REVIEW

**Review by awaelchli — APPROVED** [review]

File: examples/pl_loops/yielding_training_step.py [hunk 1]
```diff
@@ -40,7 +40,7 @@
 # code when dealing with multiple optimizers (automatic optimization).                      #
 #                                                                                           #
 # Learn more about the loop structure from the documentation:                               #
-# https://pytorch-lightning.readthedocs.io/en/stable/extensions/loops.html                  #
+# https://pytorch-lightning.readthedocs.io/en/LTS/extensions/loops.html                  #
```
Comment (awaelchli) [inline examples/pl_loops/yielding_training_step.py:43]: It actually doesn't exist (404). Is it correct?

Comment (Borda) [reply examples/pl_loops/yielding_training_step.py:43]: just uploaded :)

**Review by akihironitta — APPROVED** [review]

File: examples/README.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ ______________________________________________________________________
 
 ## Lightning Fabric Examples
 
-We show how to accelerate your PyTorch code with [Lightning Fabric](https://pytorch-lightning.readthedocs.io/en/latest/fabric/fabric.html) with minimal code changes.
+We show how to accelerate your PyTorch code with [Lightning Fabric](https://lightning.ai/docs/fabric/stable) with minimal code changes.
```
Comment (lantiga) [inline examples/README.md:16]: I saw that we are pointing to LTS in some places and to stable in others.
We should probably just point to LTS everywhere no?

Comment (Borda) [reply examples/README.md:16]: good point, so I would say if it is a specific API, then LTS in general let's point to 2.0+

**Comment by Borda** [comment]
included fix for #17199

**Review by lantiga — APPROVED** [review]

**Pull request merged**